### DISCUSSION TITLE
Sprint 3 Goals | October 11, 2022

### DISCUSSION THREAD

**Discussion by harrybsu123** [cs481-ekh/f22-ai-cbl#93] [Meeting Minutes]
Below is the actual goals.

**Comment by ninanikitina** [comment]
10/11/2022
All Team members were present

**Summary:** 
We will work on an expansion and in-depth sprint2, where we take models and estimate the outcome, then rebuild based on the feedback and ultimately optimize the general predictions. Which most likely works in the sense of back and forth until the model can predict relative satisfy output. This will be a pre-build of sprint3, which may involve an ensemble of different models. 


** Sprint 3 main goal: **

- Harry works on decision trees, where he implements two different styles, estimating regression and clarification outcomes, preparing for future ensemble models. This extends the previous decision tree study and expands to more enormous decision trees.
- Stephaine will work on a linear regression model with fewer features, and she is ambitious to work on ensemble models with techniques that include bagging, boosting, and voting. In this sense, her job will expand to the future, as linear and non-linear models will try the ensemble to get better outcomes, which is part of technical verification.
- Nina will work with Stephanie and Harry to optimize feature engineering for the linear model with approximately10 features. Most likely, she will also provide different data for the non-linear model. 



**Rotate rolls:** 

- Harry is the Scribe.     
- Nina is Project Leader.    
- Stephanie is Project Owner.
